packages/rxjs-firebase/src/operators/catchDocumentSnapshotError.ts

```
@@ -0,0 +1,32 @@
+import { type DocumentData } from '@firebase/firestore'
+import { catchError, Observable, type OperatorFunction } from 'rxjs'
```

ESLint

[error] Unable to resolve path to module 'rxjs'.

(import-x/no-unresolved)